From 269796ed52ad575fbfadce257a59534912da87c7 Mon Sep 17 00:00:00 2001
From: Damien Le Moal <damien.lemoal@opensource.wdc.com>
Date: Mon, 26 Sep 2022 20:53:05 +0000
Subject: [PATCH 14/48] scsi: Define the COMPLETED sense key

Upstream commit d9504c65be2a0794822363f3d75bb41e9c8b3691

Add the definition for the COMPLETED sense key in scsi_proto.h. While at
it, cleanup the white lines around the sense keys macro definitions.

Signed-off-by: Damien Le Moal <damien.lemoal@opensource.wdc.com>
Signed-off-by: Niklas Cassel <niklas.cassel@wdc.com>
Acked-by: Martin K. Petersen <martin.petersen@oracle.com>
---
 include/scsi/scsi_proto.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/include/scsi/scsi_proto.h b/include/scsi/scsi_proto.h
index f017843a8124..2fc8c538722e 100644
--- a/include/scsi/scsi_proto.h
+++ b/include/scsi/scsi_proto.h
@@ -208,7 +208,6 @@ enum sam_status {
 /*
  *  SENSE KEYS
  */
-
 #define NO_SENSE            0x00
 #define RECOVERED_ERROR     0x01
 #define NOT_READY           0x02
@@ -223,7 +222,7 @@ enum sam_status {
 #define ABORTED_COMMAND     0x0b
 #define VOLUME_OVERFLOW     0x0d
 #define MISCOMPARE          0x0e
-
+#define COMPLETED	    0x0f
 
 /*
  *  DEVICE TYPES
-- 
2.38.1

